cmd/telegram-butler/main.go

```diff
@@ -12,40 +13,53 @@ import (
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 
-	log := logger.NewSTDLogger()
+	log := slog.New(
```

Comment:
I can move `slog.New(...)` to the `init()` method of `logger` package if we want to)